Library/Homebrew/cmd/test-bot.rb

```diff
@@ -221,8 +221,8 @@ def initialize(argument, options={})
       @added_formulae = []
       @modified_formula = []
       @steps = []
-      @tap = options.fetch(:tap, CoreTap.instance)
-      @repository = @tap.path
+      @tap = options[:tap]
```

Comment:
The problem is hard to know what tap you are testing, supporting you want to run `brew test-bot <formula>` locally.